sys/include/embUnit/embUnit.h

```diff
@@ -35,17 +35,17 @@
 #ifndef EMBUNIT_EMBUNIT_H
 #define EMBUNIT_EMBUNIT_H
 
-#include <embUnit/Test.h>
-#include <embUnit/TestCase.h>
-#include <embUnit/TestListener.h>
-#include <embUnit/TestResult.h>
-#include <embUnit/TestSuite.h>
-#include <embUnit/TestRunner.h>
-#include <embUnit/TestCaller.h>
-#include <embUnit/RepeatedTest.h>
-#include <embUnit/stdImpl.h>
-#include <embUnit/AssertImpl.h>
-#include <embUnit/HelperMacro.h>
+#include <embUnit/Test.h> /* IWYU pragma: export */
+#include <embUnit/TestCase.h> /* IWYU pragma: export */
+#include <embUnit/TestListener.h> /* IWYU pragma: export */
+#include <embUnit/TestResult.h> /* IWYU pragma: export */
+#include <embUnit/TestSuite.h> /* IWYU pragma: export */
+#include <embUnit/TestRunner.h> /* IWYU pragma: export */
+#include <embUnit/TestCaller.h> /* IWYU pragma: export */
+#include <embUnit/RepeatedTest.h> /* IWYU pragma: export */
+#include <embUnit/stdImpl.h> /* IWYU pragma: export */
+#include <embUnit/AssertImpl.h> /* IWYU pragma: export */
+#include <embUnit/HelperMacro.h> /* IWYU pragma: export */
```

Comment:
```suggestion
#include <embUnit/Test.h>         /* IWYU pragma: export */
#include <embUnit/TestCase.h>     /* IWYU pragma: export */
#include <embUnit/TestListener.h> /* IWYU pragma: export */
#include <embUnit/TestResult.h>   /* IWYU pragma: export */
#include <embUnit/TestSuite.h>    /* IWYU pragma: export */
#include <embUnit/TestRunner.h>   /* IWYU pragma: export */
#include <embUnit/TestCaller.h>   /* IWYU pragma: export */
#include <embUnit/RepeatedTest.h> /* IWYU pragma: export */
#include <embUnit/stdImpl.h>      /* IWYU pragma: export */
#include <embUnit/AssertImpl.h>   /* IWYU pragma: export */
#include <embUnit/HelperMacro.h>  /* IWYU pragma: export */
```

Not sure if it makes sense to change that for cosmetics? 👀